demo/addons/godot-openxr/scenes/first_person_controller_vr.gd

```diff
@@ -74,13 +93,21 @@ func _get_xr_viewport() -> Viewport:
 	else:
 		return get_viewport()
 
-func _start_passthrough():
-	if start_passthrough:
-		# make sure our viewports background is transparent
-		_get_xr_viewport().transparent_bg = true
+func is_passthrough_supported() -> bool:
+	return enabled_extensions.find("XR_FB_passthrough") >= 0
```

Comment:
Is this method being used?

Reply:
Not yet, it's there for convenience, I'm assuming people who want to support this will want to check if support is available.